data/membership-schema/src/main/resources/net/corda/schema/membership/member/dynamic/registration/1.0/corda.member.dynamic.registration.json

```diff
@@ -62,6 +62,11 @@
       "examples": [
         "net.corda.notary.MyNotaryService"
       ]
+    },
+    "corda.auth.token": {
+      "description": "Optional. A UUID string that uniquely identities the pre-auth token.",
```

Comment:
extremely late on this, but I assume you meant `identifies` here?

Reply:
oh noo, but yes i meant identifies